src/common/components/select-repository-row/index.js

```diff
@@ -37,12 +47,18 @@ class SelectRepositoryRow extends Component {
   }
 }
 
+SelectRepositoryRow.defaultProps = {
+  checked: false,
```

Comment:
if the props on a form switch from undef to bool, react warns about controlled vrs uncontrolled form elements.